jwst/datamodels/schemas/lev3_prod.schema.yaml

```diff
@@ -4,238 +4,35 @@ properties:
     title: Level 3 Schema Metadata
     type: object
     properties:
-      asn:
-        title: Association Information
-        type: object
-        properties:
-          targname:
-            title: Name of target specified in ASN table
-            type: string
-            fits_keyword: ASNTARG
-          program:
-            title: Name of Program specified in ASN table
-            type: string
-            fits_keyword: ASNPROG
-          asn_type:
-            title: Type of ASN specified in ASN table
-            type: string
-            fits_keyword: ASNTYPE
-          asn_rule:
-            title: Rule used to create ASN table from ASN Pool
-            type: string
-            fits_keyword: ASNRULE 
       resample:
           title: Metadata describing resampling done using this data
           type: object
           properties:
             pointings:
-              title: Number of groups/pointings included in resampled product
+              title: Number of pointings included in resampled product
               type: integer
               fits_keyword:  NDRIZ
             product_exposure_time:
               title: Total exposure time for product
               type: number
               fits_keyword: TEXPTIME
-            product_data_extname:
-              title: Extname of SCI extension
-              type: string
-              fits_keyword: SCIEXT
-            product_context_extname:
-              title: Extname of CON extension
-              type: string
-              fits_keyword: CONEXT
-            product_weight_extname:
-              title: Extname of WHT extension
-              type: string
-              fits_keyword: WHTEXT
-            drizzle_fill_value:
-              title: Value used in pixels without any valid input data
-              type: string
-              fits_keyword: DFVAL
-            drizzle_pixel_fraction:
-              title: Drizzle pixel fraction used to create resampled product
-              type: number
-              fits_keyword: DPIXFR
-            drizzle_kernel:
-              title: Drizzle kernel used to create resampled product
-              type: string
-              fits_keyword: DKERN
-            drizzle_output_units:
-              title: Units for resampled output product as cps or counts
-              type: string
-              fits_keyword: DOUTUN
-            drizzle_weight_scale:
-              title: Drizzle weight scale used to create resampled product
-              type: string
-              fits_keyword: DWTSCL
-            resample_bits:
-              title: Bit values to consider good when interpreting input DQ arrays
-              type: integer
-              fits_keyword: RESBITS
             weight_type:
               title: Type of drizzle weighting to use in resampling input
               type: string
               enum: ['exptime','error']
-              fits_keyword: RESWHT
```

Comment:
Why remove the keyword definition for this? Are you assuming it's not something of interest to the user and hence doesn't need to be documented in a keyword?

Reply:
Well for now let's leave the keyword out, until we actually implement IVM weighting. We can put it back in then.